src/cross_section.F90

```diff
@@ -162,6 +157,7 @@ subroutine calculate_nuclide_xs(i_nuclide, i_sab, E, i_log_union, sqrtkT, &
     real(8) :: sig_t, sig_a, sig_f ! Intermediate multipole variables
 
     ! Initialize cached cross sections to zero
+    micro_xs(i_nuclide) % elastic         = -ONE
```

Comment:
Maybe we should make this default to nan and check with `isnan`.  That way, if a developer forgets to call `calculate_elastic_xs`, debug mode will hopefully help them find the error sooner.

Reply:
Yeah, minus one is probably not a good idea, but I'm not sure a NaN is either (plus, that would rely on the `IEEE_ARITHMETIC` intrinsic module which we don't presently use and, strictly speaking, is not required to be implemented -- for gfortran, it is supported only on [certain platforms](https://gcc.gnu.org/onlinedocs/gfortran/IEEE-modules.html)). I'll introduce another named parameter to handle this better.